services/system/Accounts/plugin/account_tokens/wit/world.wit

```diff
@@ -11,7 +9,15 @@ interface types {
         id: u32,
     }
 
-    type connection-token = origination-data;
+    /// Data related to the origination of the current call
+    record connection-token {
+        /// The account name of the psibase app responsible for the call.
+        /// May be empty if the call comes from an external web app.
+        app: option<string>,
+
+        /// The domain origin responsible for the call.
+        origin: string
```

Comment:
Is the origin still used anywhere?

Reply:
No, I don't believe so.

Reply:
This entire connection token object will be removed at some point soon in favor of accounts directly using `host:prompt` to trigger the redirect.